target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch

```diff
@@ -560,8 +560,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +	if (!flow)
 +		goto err_flow_alloc;
 +
-+	if (flow_offload_route_init(flow, &route) < 0)
-+		goto err_flow_add;
++	flow_offload_route_init(flow, &route);
```

Comment:
I think you have to add this part too:
````
@@ -250,9 +250,14 @@ static int nft_flow_route(const struct nft_pktinfo *pkt,
 		break;
 	}
 
+	if (!dst_hold_safe(this_dst))
+		return -ENOENT;
+
 	nf_route(nft_net(pkt), &other_dst, &fl, false, nft_pf(pkt));
-	if (!other_dst)
+	if (!other_dst) {
+		dst_release(this_dst);
 		return -ENOENT;
+	}
 
 	nft_default_forward_path(route, this_dst, dir);
 	nft_default_forward_path(route, other_dst, !dir);
````

Reply:
This should work: https://github.com/openwrt/openwrt/pull/14777/files#diff-cfaa88b234d5ef2711751b1e7a8db71489bdbfe63fd2c8a782194c07b3fc07bf

Reply:
Please review my rebase I pushed just now.

Reply:
LGTM